src/__tests__/patients/list/ViewPatients.test.tsx

```diff
@@ -80,6 +82,30 @@ describe('Patients', () => {
       expect(wrapper.find(Spinner)).toHaveLength(1)
     })
 
+    it('should render no patients exists when no patients exist', () => {
+      const wrapper = setup(false, [], 0)
+
+      const addNewPatient = wrapper.find(AddNewPatient)
+      expect(addNewPatient).toHaveLength(1)
+
+      const icon = wrapper.find(Icon).first()
+      const typography = wrapper.find(Typography)
+      const button = wrapper.find(Button)
+      const iconType = icon.prop('icon')
+      const iconSize = icon.prop('size')
+      const typographyText = typography.prop('children')
+      const typographyVariant = typography.prop('variant')
+      const buttonIcon = button.prop('icon')
+      const buttonText = button.prop('children')
+
+      expect(iconType).toEqual('patients')
+      expect(iconSize).toEqual('6x')
+      expect(typographyText).toEqual('patients.noPatients')
+      expect(typographyVariant).toEqual('h5')
+      expect(buttonIcon).toEqual('patient-add')
+      expect(buttonText).toEqual('patients.newPatient')
+    })
```

Comment:
This test is testing behavior inside the `AddNewPatient` component. I think instead, this test should test that the `AddNewPatient` component gets rendered and then this current test should be moved to an `AddNewPatient` test.